utilities/cmake/simulator/simulator.cmake

```diff
@@ -118,6 +118,10 @@ target_include_directories(${PROJECT_NAME} PRIVATE
         $<$<BOOL:UNIT_TESTS_SWITCH>:${PROJECT_SOURCE_DIR}/tests/sample_test>
```

Comment:
```suggestion
```

Reply:
we should delete inclusion of this directory in include search paths as we have deleted the directory itself
same comment applicable to firmware.cmake